### PR TITLE
#160196032 Approve/Reject Request (By Requester's Manager)

### PR DESCRIPTION
**Pull request by Nedson202**
#### What does this PR do?
- add a check to render approval buttons on My Approvals page only if requests are open
- show updated requests status on request details modal
- disable buttons when updating or when a request has been updated before

#### Description of Task to be completed?
add method to 
-  render ` Approve && Reject ` buttons only on ` My Approvals ` page
-  render buttons based on [mockup](https://projects.invisionapp.com/d/main#/console/14861108/314016836/preview) if a requests status has been updated
- disable both buttons if a requests status has been updated or isUpdating

#### How should this be manually tested?
- Clone the repo - https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Install dependencies and start the app - make start
- Create a new request on My Requests tab and ensure you're listed as your manager
- Log in, click Requests tab on the left sidebar and select My Approvals
- Click an open request
- Any button out of Approve and Reject can be clicked to update the status of the request
- Click an open request on `My Requests && My Approvals ` page to display modal
- Click on an approved or rejected request and click both buttons

#### Any background context you want to provide?
- Create a request on ` My Requests ` page and don't forget to add yourself as manager on the form
- This [PR](https://github.com/andela/travel_tool_back/pull/39/files) on the backend is required for this to work as expected.

#### What are the relevant pivotal tracker stories?
[#160196032](https://www.pivotaltracker.com/story/show/160196032)

#### Screenshots (if appropriate)
screen shot 2018-09-09 at 11 20 11 pm

#### Questions:

### CODE REVIEW

File: src/components/RequestsModal/CommentBox/__test__/AddComment.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { shallow } from 'enzyme';
 import AddComment from '../AddComment';
 
 // describe what we are testing
```
Comment (d3mola) [inline src/components/RequestsModal/CommentBox/__test__/AddComment.test.js]: @Nedson202, the test folder name should be `__tests__` instead of `__test__`

**Comment by d3mola** [comment]
@Nedson202, under the background context section, there is a part that says `explore options`, it's not clear what that means

**Comment by Nedson202** [comment]
@d3mola That has been updated please check. That was placed to show whoever is reviewing what can be tested. That has been fixed

**Comment by d3mola** [comment]
@Nedson202, as a manager, when I approve a new request, the request details modal disappears and I get redirected to the approvals page as expected but I still see the status of that request as `open` until I refresh the page. I think the status change should reflect the change of the status immediately I approve or reject the request

**Comment by Nedson202** [comment]
@d3mola I have indicated on the `background context` section that this [PR](https://github.com/andela/travel_tool_back/pull/39) on the backend is required for everything to work as expected.

File: src/components/RequestsModal/RequestInfo/__tests__/RequestInfo.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import RequestInfo from '../RequestInfo';
+import requestData from '../../__mocks__/requestData';
+
+const props = {
+  requestData,
+};
+
+// describe what we are testing
```
Comment (d3mola) [inline src/components/RequestsModal/RequestInfo/__tests__/RequestInfo.test.js]: @Nedson202 , I don't think we need this comment

Comment (Nedson202) [reply src/components/RequestsModal/RequestInfo/__tests__/RequestInfo.test.js]: This has been resolved.

**Comment by samuelkubai** [comment]
@Nedson202 please update this PR.

**Comment by Nedson202** [comment]
Yes I will. Am working on resolving the conflicts.

**Comment by samuelkubai** [comment]
@babadee001 this needs your attention

**Comment by babadee001** [comment]
LGTM @samuelkubai

**Pull request merged**